app/ui/browser-blueprint/views/overview/overview-cards.jsx

```diff
@@ -32,7 +33,12 @@ const OverviewCards = function(props) {
       {props.pages.map((page, pageIndex) => (
         <OverviewCard key={`page-${page.id}`}
           page={page}
-          pageIndex={pageIndex}
+          width={UIConstants.CARD_WIDTH}
```

Comment:
These overview cards components are inside ui/shared, intended to be shared between multiple frontends. Currently, with this PR, it's the browser-blueprint and browser-modern frontends both importing these components. The UI constants can be frontend-specific, so these cards may be distinctly themed (albeit this is only superficially at the moment).

That being said, the UI constants themselves are indeed the same between all frontends at the moment, so I can imagine having such a file specific for the cards themselves instead of relying on props passed from consumers. Would this make more sense?